index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+    <header>
+        <div class="logo-wrapper">
+            <a href="index.html" class="logo">My Logo</a>
+        </div>
+        <button type="button" class="btn-mobile-menu">
+            <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"
+                    fill="#6070FF" />
+            </svg>
+        </button>
+    </header>
+    <main>
+        <section class="headline-section">
+            <div class="section-mobile-shape"></div>
+            <div class="container">
+                <h1 class="main-greeting">
+                    <span>I’m Getinet.</span>
+                    <span>Glad to see you!</span>
+                </h1>
+                <p class="main-message">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hesitate
+                    to contact me.
+                </p>
+                <div class="social-media">
+                    <p class="connect-message">LET’S CONNECT</p>
```

Comment:
- Great job on implementing the social icons section in your code! The "LET’S CONNECT" title is a nice addition. However, it would be more semantically appropriate to use a proper heading tag for this title instead of a `<p>` tag. Consider using a `<h2>` or `<h3>` tag to indicate that it is a title. This will improve the readability and accessibility of your code. Please update your code accordingly. Keep up the good work! 👏🎉